data-sources/ajax/config.yml

```diff
@@ -0,0 +1,27 @@
+id: ajax
```

Comment:
Let's remove this data source in favor of the existing `new-relic-browser` data source which we can link to in the quickstart config. This data source provides a link to a resource that is more about using the data from the agent than installing it, so let's make sure this docs link is represented in the quickstart description or as a documentation item.